frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -76,6 +80,13 @@ pods:
         goal: FINISHED
         cmd: ./apache-cassandra-{{CASSANDRA_VERSION}}/bin/nodetool snapshot -p {{TASKCFG_ALL_JMX_PORT}} -t $CASSANDRA_SNAPSHOT_NAME -- $CASSANDRA_KEYSPACES
         resource-set: sidecar-resources
+      upload-aws:
+        goal: FINISHED
+        cmd: >
+                chmod a+x {{S3CLI_VERSION}} && eval "echo \"$(sed 's/\"/\\\"/g' < s3config.json)\"" > config.json
```

Comment:
I like the JSON in a separate file.  The command isn't too beautiful though.  I'm trying to say something constructive, but I don't have an elegant solution handy, just think we could find a better way if we thought about it a bit.